src/main/java/bio/terra/app/utils/startup/StartupInitializer.java

```diff
@@ -13,6 +13,5 @@ private StartupInitializer() {}
   public static void initialize(ApplicationContext applicationContext) {
     // Initialize jobService, stairway, migrate databases, perform recovery
     JobService jobService = (JobService) applicationContext.getBean("jobService");
-    jobService.initialize();
```

Comment:
If we call `initialize()` from the constructor I'm now thinking this class isn't necessary at all. Might be worth removing it and seeing if things break.

Reply:
That's a good point! I'll try that out

Reply:
Looking at the code, the intent is that the database migrations are run before the server starts up, and this is used by code in `ApplicationConfiguration` to ensure that the migrations are done at the right time. I still wonder if it's necessary though. But maybe that kind of change shouldn't be part of this PR.